devito/dse/rewriters.py

```diff
@@ -369,6 +375,48 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
         return cluster.rebuild(processed)
 
+    @dse_pass
+    def _skewing(self, cluster, template, **kwargs):
+        """
+        Skew the accesses along the time Dimension.
+        """
+
+        skew_factor = self.DEFAULT_SKEW_FACTOR
+
+        _, sub_iterators, directions = cluster.ispace.args
+
+        skew_dim, mapper, intervals = None, {}, []
```

Comment:
if i may suggest, i think `skew_dim` doesn't need to be initialized

also, maybe `mapper` and `intervals`' initialization could come "logically before" it's utilization, like before lines 399